src/test/frontend/handler/editor_test.cljs

```diff
@@ -45,56 +44,6 @@
           "[[https://github.com/logseq/logseq][logseq]] is #awesome :)" 0 editor/url-regex))
       "Finds url in org link correctly"))
 
-(defn- keydown-not-matched-handler
-  "Spied version of editor/keydown-not-matched-handler"
-  [{:keys [value key format cursor-pos] :or {key "#" format "markdown"}}]
-  ;; Reset editor action in order to test result
-  (state/set-editor-action! nil)
-  ;; Default cursor pos to end of line
-  (let [pos (or cursor-pos (count value))]
-    (with-redefs [util/get-selected-text (constantly false)
-                  state/get-input (constantly #js {:value value})
-                  cursor/pos (constantly pos)
-                  cursor/get-caret-pos (constantly {})]
-      ((editor/keydown-not-matched-handler format)
-       #js {:key key} nil))))
-
-(deftest keydown-not-matched-handler-test
```

Comment:
Given the number of bugs we've had in autocomplete, these tests are helpful, especially since they test scenarios the new test doesn't. Is there a reason we can't convert these to use a mocked keyup-handler? I'm happy to help if needed

Reply:
It is now handled by `handle-last-input`.
The test cases can be written in `handle-last-input-handler-test`. I'll address this.